app/controllers/products_controller.rb

```diff
@@ -8,29 +9,14 @@ def show
   end
 
   def index
-    @products = Product.search(params[:q].downcase) if params[:q].present?
-    @products ||= Product.all
-    handle_select_category
-    handle_sort
-    @products = @products.paginate(page: params[:page])
+    @q = Product.ransack(params[:q])
+    @q = Product.ransack(params[:q][:category_id]) if params[:q][:category_id]
```

Comment:
line 12 13 em dùng toán tử 3 ngôi đi nhé

Reply:
dùng toán tử 3 ngôi thì bị rubocop bắt lỗi, line too long nên e dùng if else nha chị